src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            String[] temporaryArray;
+            int supplyInt = 0;
+            int buyInt = 0;
+            while ((line = bufferedReader.readLine()) != null) {
+                temporaryArray = line.split(",");
+                if (temporaryArray[0].equals("supply")) {
+                    supplyInt += Integer.parseInt(temporaryArray[1]);
+                } else if (temporaryArray[0].equals("buy")) {
+                    buyInt += Integer.parseInt(temporaryArray[1]);
+                }
```

Comment:
Instead of using direct String concatenation in the loop, use a StringBuilder to improve performance and reduce the number of created String objects.